engine-rest/engine-rest-openapi/src/main/templates/paths/history/batch/cleanable-batch-report/count/get.ftl

```diff
@@ -0,0 +1,53 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/batch/get-cleanable-batch-report-count/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getCleanableHistoricBatchesReportCount"
+      tag = "Historic Batch"
+      summary = "Get Cleanable Batch Report Count"
+      desc = "Queries for the number of report results about a historic batch operations relevant
+              to history cleanup (see
+              [History cleanup](${docsUrl}/user-guide/process-engine/history/#history-cleanup)
+              ).
+              Takes the same parameters as the
+              [Get Cleanable Batch Report](${docsUrl}/reference/rest/history/batch/get-cleanable-batch-report/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign last = false >
+    <#include "/lib/commons/sort-params.ftl">
```

Comment:
Do we need to add that this can be sorted by `finished`?

Reply:
yes

Reply:
Here not necessarily as it is a count method. Thus, sorting does not have any effect anyways.